docs/ossm/ossm2-migration/multi-tenancy/README.md

```diff
@@ -0,0 +1,285 @@
+       openshiftRoute:
+         enabled: false
+     mode: MultiTenant
+     policy:
+       type: Istiod
+     profiles:
+       - default
+     telemetry:
+       type: Istiod
+     tracing:
+       type: None
+     version: v2.6
+   ```
+
+1. Create bookinfo namespace
+
+   ```sh
+   oc create ns bookinfo
+   ```
+
+1. Add bookinfo to the SMMR
+
+   ```yaml
+   apiVersion: maistra.io/v1
+   kind: ServiceMeshMemberRoll
+   metadata:
+     name: default
+     namespace: tenant-a
+   spec:
+     members:
+       - bookinfo
+   ```
+
+1. Deploy bookinfo
+
+   ```sh
+   oc apply -n bookinfo -f https://raw.githubusercontent.com/Maistra/istio/maistra-2.6/samples/bookinfo/platform/kube/bookinfo.yaml
+   ```
+
+1. Ensure pods are all healthy and you see `2/2` pods indicating a sidecar was injected.
+
+   ```sh
+   oc get pods -n bookinfo
+   ```
+
+   Example output
+
+   ```sh
+   NAME                             READY   STATUS    RESTARTS   AGE
+   details-v1-75cb5b97b4-5c6nm      2/2     Running   0          6h13m
+   productpage-v1-899d756d8-ch424   2/2     Running   0          6h10m
+   ratings-v1-58757c649b-8bdg4      2/2     Running   0          6h13m
+   reviews-v1-6878c868b6-42kw7      2/2     Running   0          6h13m
+   reviews-v2-6c8bd45654-jpt76      2/2     Running   0          6h13m
+   reviews-v3-57997d6ccd-j6pmh      2/2     Running   0          6h13m
+   ``` -->
+
+1. Create your `Istio` resource.
+
+   Here we are setting `discoverySelectors` on our `Istio` resource. In 3.x, controlplanes by default watch the entire cluster and when managing multiple controlplanes on a single cluster, you must narrow the scope of each controlplane by setting `discoverySelectors`. In this example, we use the label `tenant` but you can use any label or combination of labels that you choose.
+
+   **Note:** it is important your `Istio` resource's `spec.namespace` field is the **same** namespace as your `SMCP`. If you set your `Istio` resource's `spec.namespace` field to a different namespace than your `SMCP`, the migration will not work properly. In this example, we assume that your `SMCP` is found in the `tenant-a` namespace.
+
+   ```yaml
+   apiVersion: sailoperator.io/v1alpha1
+   kind: Istio
+   metadata:
+     name: istio-tenant-a
+   spec:
+     namespace: tenant-a
+     values:
+       meshConfig:
+         discoverySelectors:
+           - matchLabels:
+               tenant: tenant-a
+     version: v1.23.0
+   ```
+
+1. Add your `tenant` label to each one of your dataplane namespaces.
+
+   With 2.x, we enrolled namespaces into the mesh by adding them to the `SMMR` resource. In 3.x, you must label each one of your dataplane namespaces with this label. For every namespace in your SMMR, add your tenant label to the namespace.
+
+   ```sh
+   oc label ns bookinfo tenant=tenant-a
+   ```
+
+   Now we are ready to migrate our workloads from our 2.x controlplane to our 3.x controlplane.
+
+1. Update injection labels on the dataplane namespace.
```

Comment:
If you add a "migrate workloads" heading, I'd probably start it here, because once you update these labels, it sounds like you're committing the workloads to move to the new mesh at next restart.